coral/src/app/layout/MainNavigation.test.tsx

```diff
@@ -0,0 +1,67 @@
+import MainNavigation from "src/app/layout/MainNavigation";
+import { cleanup, screen, render, within } from "@testing-library/react";
+
+// mock out svgs to avoid clutter
+jest.mock("@aivenio/design-system", () => {
```

Comment:
How it is clutter? Is the `screen.logTestingPlaygroundURL` hard to read while creating / debugging the tests? If we agree so, we could make it a global mock and preferrably would include the icon name in the `data-testid` if possible.

Reply:
Yes, `screen.debug()` (which is also printed in error cases) is getting really long because it prints out the whole svg. I wanted to move that in the setupAfterEnv in a different PR, because this one is already so big 🙈

Reply:
created a follow up issue https://github.com/aiven/klaw/issues/282